turbopuffer/__init__.py

```diff
@@ -2,7 +2,7 @@
 import sys
 api_key = os.environ.get('TURBOPUFFER_API_KEY')
 api_base_url = os.environ.get('TURBOPUFFER_API_BASE_URL', 'https://api.turbopuffer.com')
-upsert_vectors_as_base64 = os.environ.get('TURBOPUFFER_UPSERT_VECTORS_AS_BASE64', 'true').lower() == 'true'
+encode_vectors_as_base64 = os.environ.get('TURBOPUFFER_ENCODE_VECTORS_AS_BASE64', 'true').lower() == 'true'
```

Comment:
This is technically a breaking change, though I don't think anyone's currently using the parameter.